AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.css

```diff
@@ -38,6 +38,17 @@
     width: 15%;
 }
 
+.vh-image-10 {
+    width: 15%;
+    padding-left: 10%;
+}
+
+.vh-no-image {
+    float: left;
```

Comment:
Consider using flex layouts here instead of floats. Also, with margins & padding, it might be best to use `em` instead of percentages because these will scale based on the users' text size

Reply:
updated to flex layout.